pyuvdata/__init__.py

```diff
@@ -6,28 +6,32 @@
 
 """
 from __future__ import absolute_import, division, print_function
+import warnings
+from pkg_resources import get_distribution, DistributionNotFound
+
+# Set the version automatically from the package details.
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
these are the lines that require the package not to be installed right? Could noqa them since I can't think of a reasonable way to even test it :thinking:

Reply:
Good idea.